README.md

```diff
@@ -36,19 +34,26 @@ sma = tp.simple_moving_average(node, window_length=tp.days(7))
 result = sma.evaluate(evset)
```

Comment:
What about adding a print and plot?

>> tp.read_event_set("path/to/temporal_data.csv", timestamp_column="time")
What about showing / describing what the csv file contains. Alternatively, what about creating an example by hand.